### PR TITLE
DevEx: Moved COMMON_CITIES, SMALL_CITIES, TRIAL_CITIES, TRIAL_CITY_STRINGS, SESSION_TERMS, SESSION_TYPES

### PR DESCRIPTION
**Pull request by ericsorenson**

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]

File: shared/src/persistence/s3/getUploadPolicy.js [hunk 1]
```diff
@@ -1,6 +1,3 @@
-exports.MAX_FILE_SIZE_MB = 250; // megabytes
-exports.MAX_FILE_SIZE_BYTES = exports.MAX_FILE_SIZE_MB * 1024 * 1024; // bytes -> megabytes
```
Comment (kkoskelin) [inline shared/src/persistence/s3/getUploadPolicy.js:2]: seems this one is needed below in the headers...?

File: web-api/src/applicationContext.js [hunk 2]
```diff
@@ -1057,7 +1055,7 @@ module.exports = (appContextUser = {}) => {
         status => status !== CASE_STATUS_TYPES.closed,
       ),
       ORDER_TYPES_MAP: ORDER_TYPES,
-      SESSION_STATUS_GROUPS: TrialSession.SESSION_STATUS_GROUPS,
+      SESSION_STATUS_GROUPS: SESSION_STATUS_GROUPS,
```
Comment (kkoskelin) [inline web-api/src/applicationContext.js:1058]: `SESSION_STATUS_GROUPS,`

**Review by kkoskelin — CHANGES_REQUESTED** [review]
Just one bit that I think needs more attention for a max file size header.

**Review by kkoskelin — APPROVED** [review]

**Review by sutt0n — APPROVED** [review]

**Pull request merged**